path_config_test.go

```diff
@@ -195,6 +195,27 @@ func TestConfig(t *testing.T) {
 		t.Fatalf("got unexpected error: %v", resp.Error())
 	}
 
+	// test invalid CA cert
+	data = map[string]interface{}{
+		"kubernetes_ca_cert": "bad",
```

Comment:
nit: it would be good if we had a more realistic certificate here. Perhaps one that is truncated?